### PR TITLE
update avatar - add option clickable

### PR DESCRIPTION
**Pull request by aju**
screenshot 2018-11-13 at 14 26 45
Needed for SEO improvements ! :)

### CODE REVIEW

File: src/components/avatar/Avatar.jsx [hunk 1]
```diff
@@ -1,27 +1,48 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+// @flow
```
Comment (burakukula) [inline src/components/avatar/Avatar.jsx]: ❤️

File: src/components/avatar/Avatar.jsx [hunk 2]
```diff
@@ -1,27 +1,48 @@
...
   XXLARGE: {className: 'xxlarge', iconSize: 96},
   XXXLARGE: {className: 'xxxlarge', iconSize: 120}
-};
+});
 
-const Avatar = ({size = SIZE.NORMAL, border = false, spaced, imgSrc, className, link, title, ...props}) => {
+const Avatar = ({
+  size = SIZE.NORMAL,
+  border = false,
+  spaced, imgSrc,
```
Comment (burakukula) [inline src/components/avatar/Avatar.jsx]: @aju separate line please

Comment (aju) [reply src/components/avatar/Avatar.jsx]: OMG yes :D

**Comment by burakukula** [comment]
@aju keep in mind accessibility, maybe `aria-label` as an option here?

**Comment by burakukula** [comment]
@aju https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_link_role

**Comment by aju** [comment]
After thinking about it I think clickable should not be on avatar itself. I will close this one.

**Pull request closed without merging**